tools/etl/tg-jdbc-driver/README.md

```diff
@@ -10,6 +10,7 @@ The TigerGraph JDBC Driver is a Type 4 driver, converting JDBC calls directly in
 | 1.2.2 | 2.4.1~3.4 | 1.8 | Rest++ | ResultSet | Bug fix |
```

Comment:
I guess we could remove the "Protocol" and "Query Result Format" columns. They never changed. The reason for adding the "Protocol" column in the very beginning, is that we planned there would be a new release to support talking with gsql_server directly.
For the "Query Result Format" column, that's because previously we only support outputting JSON format.

Reply:
Ok, will do that in next PR